corehq/apps/commtrack/processing.py

```diff
@@ -125,7 +125,7 @@ def process_stock(xforms, case_db=None):
 
     stock_report_helpers = []
     case_action_intents = []
-    sorted_forms = sorted(xforms, key=lambda f: 0 if f.is_deprecated else 1)
+    sorted_forms = sorted(xforms, key=lambda f: 1 if f.is_deprecated else 0)
```

Comment:
ordering change her is so that we process the new form first. Nothing really depends on this ordering